src/engines/html5/html5.js

```diff
@@ -69,6 +85,7 @@ export default class Html5 extends FakeEventTarget implements IEngine {
    */
   constructor(source: Source, config: Object) {
     super();
+    this._config = config;
     this._eventManager = new EventManager();
     this._createVideoElement();
     this._loadMediaSourceAdapter(source, config);
```

Comment:
If we already save the config on the _this_ we don't need to send it to `_loadMediaSourceAdapter `